zkevm-circuits/src/evm_circuit/execution/return_revert.rs

```diff
@@ -97,6 +97,15 @@ impl<F: Field> ExecutionGadget<F> for ReturnRevertGadget<F> {
             is_create.clone() * is_success.expr() * not::expr(copy_rw_increase_is_zero.expr());
         let (caller_id, address, reversion_info, code_hash) =
             cb.condition(is_contract_deployment.clone(), |cb| {
+                // EIP-211 RETURNDATASIZE should be 0 for CREATE/CREATE(2)
+                for (field_tag, value) in [
+                    (CallContextFieldTag::LastCalleeReturnDataOffset, 0.expr()),
+                    (CallContextFieldTag::LastCalleeReturnDataLength, 0.expr()),
+                ] {
+                    // let cell = self.query_cell_with_type(phase);
```

Comment:
leftover?

Reply:
y! just clean it and thanks for the catch up!